sound/soc/sof/intel/hda-dai.c

```diff
@@ -706,8 +706,7 @@ static const struct snd_soc_dai_ops ipc3_ssp_dai_ops = {
 	.shutdown = ssp_dai_shutdown,
 };
 
-static int ipc4_be_dai_trigger(struct snd_pcm_substream *substream,
-			       int cmd, struct snd_soc_dai *dai)
+static int ipc4_be_common_trigger(struct snd_soc_dai *dai, int cmd, int stream)
```

Comment:
nit-pick: i would keep the ipc4_be_dai_ prefix to show this is a DAI operation. a BE is a dailink. We keep making the confusion....

Reply:
> nit-pick: i would keep the ipc4_be_dai_ prefix to show this is a DAI operation. a BE is a dailink. We keep making the confusion....

Updated.